spec/granite/validations/validation_helpers_spec.cr

```diff
@@ -234,4 +244,26 @@ describe Granite::ValidationHelpers do
       lengthTest.errors[1].message.should eq "description is too long. It must be at most 25"
     end
   end
+
+  context "Uniqueness" do
+    it "should work for uniqueness" do
+      personUniqueness1 = PersonUniqueness.new
+      personUniqueness2 = PersonUniqueness.new
+
+      personUniqueness1.name = "awesomeName"
+      personUniqueness2.name = "awesomeName"
+
+      personUniqueness1.save
+      personUniqueness2.save
+
+      personUniqueness1.errors.size.should eq 0
+      personUniqueness2.errors.size.should eq 1
+
+      personUniqueness2.errors[0].message.should eq "name should be unique"
+
+      # Should be valid because it should not check uniqueness on itself
```

Comment:
Can you make a separate test for this use case?

Reply:
Ofcourse